Sources_v3/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -13,6 +13,23 @@ open class ChatChannelListVC<ExtraData: UIExtraDataTypes>: UIViewController,
     public var controller: _ChatChannelListController<ExtraData>!
     public var uiConfig: UIConfig<ExtraData> = .default
     public var didSelectChannel: (_ChatChannel<ExtraData>) -> Void = { _ in }
+    public var didSelectCurrentUser: () -> Void = {}
+    public var didSelectCreateNewChannel: () -> Void = {}
```

Comment:
I hope we'll move to delegates as the project matures, to be both inline with LLC and UIKit

Reply:
The thing is that are not plain callbacks but the navigation. I tried to incapsulate it in a single object of `ChatChannelListRouter` type. Will be happy to listen to your thoughts on it 🙂